en/appendices/3-4-migration-guide.rst

```diff
@@ -122,6 +122,18 @@ Cake\Validation\Validator
     * ``provider()``
 Cake\View\StringTemplateTrait
     * ``templates()``
+Cake\View\ViewBuilder
+    * ``templatePath()``
+    * ``layoutPath()``
+    * ``autoLayout()``
```

Comment:
Might be better to specifically mention the new methods in the section above as this method doesn't folllow the pattern used here.